### ISSUE TITLE
help understanding

### ISSUE THREAD

**Issue by fritjofkl** [Akylas/oss-weather#283]
Hi, super app. What means the graph beside the cloudsymbol, lines heavy...  and numbers in Meter?

**Comment by farfromrefug** [comment]
@fritjofkl not sure what you are refering To. Can you share a screenshot ?

**Comment by fritjofkl** [comment]
Yes: See the graphics left of the cloudsymbol: lines and the items 0m-10m-20m-30m and a blue curve. 
Greetings

Am 23. Dezember 2024 08:50:58 MEZ schrieb farfromrefuge ***@***.***>:

**Comment by farfromrefug** [comment]
@fritjofkl  OK I see it is the minutely raining forecast

**Comment by fritjofkl** [comment]
Thank you. Great app.

Am 23. Dezember 2024 17:15:01 MEZ schrieb farfromrefuge ***@***.***>:

**Comment by 0-5-0** [comment]
This issue can be closed?
